wordpress-mobile/AztecEditor-iOS
# Update project to have iOS 11 has a minimum

This just updates the project to assume iOS 11 has a minimum.

To test:
 - Check if the project compiles correctly and unit tests are good.
 - Some test some of the demo content to see if all is ok.
